src/components/HitsPerPage.vue

```diff
@@ -43,7 +44,7 @@ export default {
       }
     ),
     createPanelConsumerMixin({
-      mapStateToCanRefine: state => state.hasNoResults === false,
+      mapStateToCanRefine: state => Boolean(state.canRefine),
```

Comment:
Isn't `state.canRefine` already a boolean?

Reply:
Not always, it might be `undefined`, see here : https://github.com/algolia/vue-instantsearch/pull/876
Should I maybe mention it in a comment or would it be redundant ?

Reply:
I see, thanks! I don't think there is a need to add a comment in that case.